hw2/test-part

```diff
@@ -5,26 +5,27 @@ DIR="$1"
 PART="$2"
 # From https://stackoverflow.com/questions/29832037/how-to-get-script-directory-in-posix-sh
 HERE=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+FILE_PREFIX="${HERE}/${PART}"
```

Comment:
Why not put the `out` part into the `FILE_PREFIX`? Potentially even just have `OUTPUT_FILE` and for later homeworks, `SEXP_FILE` and `PP_FILE`?